### DISCUSSION TITLE
Find all broken wikilinks

### DISCUSSION THREAD

**Discussion by kukimik** [srid/emanote#192] [Ideas]
It could be useful to have an index page that shows all pages that contain broken wikilinks (i.e. links that lead to the "Missing link" page). (Or, instead of an index, another way to find them.)

Perhaps it would be even better to list all missing targets of wikilinks, and then the pages linking to them could be found using the backlinks feature.

**Comment by kukimik** [comment]
There's now issue #246 that addresses this.

**Comment by vcavallo** [comment]
I'm importing _some_ of a very large collection of notes into a directory that will be built with emanote. The strategy is to add tags to notes that I want to skip importing (this is handled in a python script). I'm expecting very many broken links and I'm okay with this. Any links to these unimported private notes will remain in the source (an obsidian vault that I don't want to monkey with) and simply be broken in the generated site.

I recognize that as of _now_, `gen` will fail on broken links. What I'm getting to is requesting one or all of the following options which would allow me to use emanote for my use case:

- allow broken links in generated sites
- elide broken links during site generation (this seems infeasible)

**Reply by srid** [reply]
This should work:

```
emanote --allow-broken-links gen /tmp/output
```

Is there anything else, in particular, you are requesting? I didn't understand the 2nd point.

**Reply by vcavallo** [reply]
that works, yea, thanks!

As to the second point: I was suggesting an option that removes the broken links from the built files. But it's kind of a weird option and probably not worth it.
